### PR TITLE
Add privacy text to SDK download pages

### PR DESCRIPTION
**Pull request by kwalrath**
Staged:
https://kw-www-dartlang-1.firebaseapp.com/tools/sdk/archive
https://kw-www-dartlang-1.firebaseapp.com/tools/sdk#install

Fixes dart-lang/site-www#968

### CODE REVIEW

File: src/tools/sdk/index.md [hunk 1]
```diff
@@ -30,12 +30,15 @@ NOTE to editors: Keep the zip file link as the last thing in the paragraph,
...
 {% endcomment %}
 
-{% if site.data.pkg-vers.SDK.channel == 'dev' -%}
 <aside class="alert alert-warning" markdown="1">
-  **Important:** The **Dart 2** SDK is available from the **dev channel** only.
-</aside>
+  **Important:**
+{% if site.data.pkg-vers.SDK.channel == 'dev' -%}
+  The **Dart 2** SDK is available from the **dev channel** only.
 {% endif -%}
```
Comment (chalin) [inline src/tools/sdk/index.md]: You should be able to indent the `{% if ... %}` and `{% endif %}`.

**Comment by chalin** [comment]
@kwalrath - can we merge this or are you waiting for @JekCharlsonYu?

**Comment by kwalrath** [comment]
I'm going to close this, but I'll probably do a similar PR in the future.

**Comment by JekCharlsonYu** [comment]
Pending from another person - there's a thread on that. She will be
updating soon.

**Pull request closed without merging**